mlrun/datastore/datastore.py

```diff
@@ -51,7 +51,10 @@ def parse_url(url):
 
 def schema_to_store(schema):
     # import store classes inside to enable making their dependencies optional (package extras)
-    if not schema or schema in ["file"] + list(string.ascii_lowercase):
+
+    # The expression `list(string.ascii_lowercase)` generates a list of lowercase alphabets,
+    # which corresponds to drive letters in Windows file paths such as `C:/Windows/path`.
```

Comment:
Ok, now you're making me worry... Should it be lowercase or uppercase? 😬

Reply:
Like you said, the schema (or in this case drive letter) is always parsed as a low case. So when C:/Windows path is parsed, the schema will be 'c' and not 'C'. No worries :)

Reply:
I just found the discrepancy a bit confusing.